### DISCUSSION TITLE
Suggestion: add wind generators and "wind system"

### DISCUSSION THREAD

**Discussion by Uran-228** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1646] [General]
I propose to add a new source of energy: a wind generator - it would be possible to add two types of wind turbines: the first one that is installed in residential buildings (small in size) and industrial (large or giant).
![IMG_1450](https://github.com/user-attachments/assets/00316658-cb21-4521-a6c5-d67e1b55df30)
![IMG_1451](https://github.com/user-attachments/assets/5989d4dc-1126-4335-8f0a-cca62ee8800f)
2 examples residental buildings wind generators

![IMG_1449](https://github.com/user-attachments/assets/a7853b83-8821-4be3-a529-3986199d8be0)
And here there are industrial wind generators

It is clear that wind is needed to extract energy from them, but there is no wind in minecraft, I could suggest how the wind can work in the game.The most probably simple wind system is from the height Y, the higher, the stronger the wind (more energy) and if the Y is less, then the wind will be weaker (it was possible to make it so that from a certain height there is no wind) .And it will also depend on the weather, if it rains, then the wind will be a little stronger, if there is a thunderstorm, then the wind will be even stronger.
And if you'll be accept my idea, can I about their craft. Can you include aluminium in craft pls :)

I like this mod and I wish you success and development, thanks for reading this suggestion. :-D
GOOD LUCK!  😎

**Comment by BallOfEnergy1** [comment]
Hey uh, suggestions aren't really allowed here.

**Reply by BallOfEnergy1** [reply]
Oh this isn't a GitHub issue, Gmail lied to me.

**Comment by JameH2** [comment]
I would suggest making it require engine lubricant to even work

**Reply by Uran-228** [reply]
I support your suggestion :D

**Comment by HbmMods** [comment]
![grafik](https://github.com/user-attachments/assets/364f1626-f286-42a5-98d1-6b9a3ea240b1)
